hieradata_aws/class/integration/signon_db_admin.yaml

```diff
@@ -1,4 +1,16 @@
 govuk_env_sync::tasks:
+  "pull_signon_staging_daily":
+    ensure: "present"
+    hour: "2"
+    minute: "30"
+    action: "pull"
+    dbms: "mysql"
+    storagebackend: "s3"
+    database: "signon_production"
+    database_hostname: "signon-mysql"
+    temppath: "/tmp/signon_staging"
+    url: "govuk-staging-database-backups"
+    path: "signon-mysql"
```

Comment:
Ah, Signon has a separate model, it doesn't get overwritten in Integration. We should remove these lines.